README.md

```diff
@@ -1,104 +1,130 @@
-# nucypher-ts
+ A TypeScript library for interacting with access control functionality in the browser.
 
-Communicate with NuCypher network from browser.
+ Full Documentation can be found [here](https://docs.threshold.network/app-development/threshold-access-control-tac).
 
-## Disclaimer
+> :warning: 
+> `nucypher-ts` is under [active development](https://github.com/nucypher/nucypher-ts/pulls).
+>
+> SDK does not support policy revocation.
+>
+> We expect breaking changes.
+# Get Started with TAC
 
-**This is a work in progress**
+The following interactive tutorial is a quick way for developers to familiarize themselves with the Threshold Access Control service. Before jumping into the following, we recommend reading this introduction to [Conditions Based decryption](https://docs.threshold.network/fundamentals/threshold-access-control/conditions-based-decryption-cbd) and its key concepts. CBD is the primary technology underpinning Threshold Access Control and directly enables data sharing predicated on the fulfillment of predefined conditions.
 
-- SDK does not support policy revocation.
-- We expect breaking changes.
+## 1. Install nucypher-ts
 
-## Supported networks
+We'll kick things off by installing `nucypher-ts` – a TypeScript library for interacting with access control functionality in the browser. The APIs for leveraging most TAC functionality are contained in nucypher-ts.
```

Comment:
Is it "most TAC fuctionality" or all of it. What's missing?

```suggestion
We'll kick things off by installing `nucypher-ts` – a TypeScript library for employing access control functionality in the browser. The APIs for leveraging TAC functionality are contained in nucypher-ts.
```

Reply:
not sure, i copied this from the gitbook